pages/vi/profiles/inDepthh.md

```diff
@@ -0,0 +1,32 @@
+# Hi,
+
+### My name is Ryan Couture, I live in the United States of America (Eastern Standard time - UTC-05:00)
+
+
+*OS: Windows 10 Pro Version 1803*
+
+	  Description: I am a passionate Software Development student at NHTI.
+	  I am graduating soon and hope to find a job after. This internship will
+	  give me a lot of valuable experience to get a job in IT.
+
+**I have good experience with Languages such as**
+
+<ul>
```

Comment:
We don't want html tag used here. For bullet problem seems like you didn't added line below **I have good....** so rawgit was not able to render it.
So please do add one line break before you start bullet. Also you can try `-` instead of `*`. Both of these requires line break before bullet is started.

Reply:
It seems to be working well with the line break. A mistake that will not be made again, thanks.